src/modal/modal.component.ts

```diff
@@ -141,7 +142,7 @@ export class Modal implements AfterViewInit, OnChanges, OnDestroy {
 	/**
 	 * Size of the modal to display.
 	 */
-	@Input() size: "xs" | "sm" | "lg";
+	@Input() size: "xs" | "sm" | "md" | "lg";
```

Comment:
```suggestion
	@Input() size: "xs" | "sm" | "md" | "lg" = "md";
```

The default size of modal is md (`carbon-components` indirectly sets the size to be `md`).

Reply:
solved on [a28466b](https://github.com/carbon-design-system/carbon-components-angular/pull/2365/commits/a28466bd8b69484176b87c3d52374ed52d06a3ec)